subsys/net/ip/dhcpv4.c

```diff
@@ -1044,7 +1044,7 @@ static enum net_verdict net_dhcpv4_input(struct net_conn *conn,
 	 */
```

Comment:
I hope it is ok, that I comment here.

The comment here does not seem accurate anymore? Can it be removed?

Reply:
Good catch, I've fixed the comment as well.

Reply:
Thanks :+1: